### PR TITLE
COMMERCE-4565 Move Commerce sections outside Control Panel

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:sf

File: modules/apps/commerce/BREAKING_CHANGES.markdown [hunk 1]
```diff
@@ -341,7 +341,28 @@ and want users to be able to sort products.
...
+
+### Concurrency Management for Inventory
+- **Date:** 2020-Sep-3
+- **JIRA Ticket:** [COMMERCE-4565](https://issues.liferay.com/browse/COMMERCE-4565)
+
+#### What changed?
+
+Moved Commerce at the same level of Applications and Control Panel in the
+Applications Menu. Also dropped the commerce-admin-api and commerce-admin-web
+modules.
```
Comment (dsitu) [inline modules/apps/commerce/BREAKING_CHANGES.markdown:354]: @Commando18, I think this sounds better.

```suggestion
Moved Commerce to the same level as Applications and Control Panel in the
Applications Menu. Also dropped the commerce-admin-api and commerce-admin-web
modules.
```

File: modules/apps/commerce/BREAKING_CHANGES.markdown [hunk 2]
```diff
@@ -341,7 +341,28 @@ and want users to be able to sort products.
...
+
+#### Who is affected?
+
+Developers: It's not possible anymore to add items to Commerce->Settings section
+by implementing CommerceAdminModule.
+
+End Users: The Commerce sections are now at the same level of the Control Panel.
```
Comment (dsitu) [inline modules/apps/commerce/BREAKING_CHANGES.markdown:361]:
```suggestion
End Users: The Commerce sections are now at the same level as the Control Panel.
```

File: modules/apps/commerce/BREAKING_CHANGES.markdown [hunk 3]
```diff
@@ -341,7 +341,28 @@ and want users to be able to sort products.
...
+
+#### Why was this change made?
+
+This change was made in order not to lose the navigation scope in the header
+bar, while remaining compliant with DXP 7.3 standards.
```
Comment (dsitu) [inline modules/apps/commerce/BREAKING_CHANGES.markdown:366]:
```suggestion
bar while remaining compliant with DXP 7.3 standards.
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by Commando18** [comment]
@dsitu I made all the requested changes, thanks!

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:sf

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**